src/explore-education-statistics-frontend/src/services/sitemapService.ts

```diff
@@ -0,0 +1,141 @@
+import methodologyService, {
+  MethodologySitemapSummary,
+} from '@common/services/methodologyService';
+import publicationService, {
+  PublicationSitemapSummary,
+} from '@common/services/publicationService';
+import dataSetService, {
+  DataSetSitemapSummary,
+} from '@common/services/dataSetService';
+import { ISitemapField } from 'next-sitemap';
+
+export default async function getSitemapFields(): Promise<ISitemapField[]> {
+  const methodologies = await methodologyService.getSitemapSummaries();
+  const publications = await publicationService.getSitemapSummaries();
+  const dataSets = await dataSetService.getSitemapSummaries();
+
+  const sitemapFields: ISitemapField[] = [
+    // matches dynamic routes following the patterns:
+    //  methodology/[methodology]
+    ...buildMethodologyRoutes(methodologies),
+
+    // matches dynamic routes following the patterns:
+    //  data-catalogue/[publicationSlug]
+    //  data-tables/[publicationSlug]
+    //  find-statistics/[publication]
+    //  find-statistics/[publication]/data-guidance
+    //  find-statistics/[publication]/prerelease-access-list
+    //  find-statistics/[publication]/[release]/data-guidance
+    //  find-statistics/[publication]/[release]/prerelease-access-list
+    //  data-catalogue/[publicationSlug]/[releaseSlug]
+    //  data-tables/[publicationSlug]/[releaseSlug]
+    //  find-statistics/[publication]/[release]
+    ...buildPublicationRoutes(publications),
+
+    // matches dynamic routes following the patterns:
+    //  data-catalogue/data-set/[dataSetFieldId]
+    ...buildDataSetRoutes(dataSets),
```

Comment:
Would comment this out for the time being and leave a TODO.

These paths are still in development and shouldn't be indexed right now.